### ISSUE TITLE
"legendary launch" "expected one argument" error

### ISSUE THREAD

**Issue by NerosTie** [Heroic-Games-Launcher/HeroicGamesLauncher#2895]
### Describe the bug

With the 2.9.0 update I can't launch games anymore.

I have this error in the log:

> legendary launch: error: argument --wrapper: expected one argument

It was ok before the update.

### Add logs

```text
Game Settings: {
	"autoInstallDxvk": true,
	"autoInstallVkd3d": true,
	"preferSystemLibs": false,
	"enableEsync": true,
	"enableFsync": true,
	"nvidiaPrime": false,
	"enviromentOptions": [],
	"wrapperOptions": [],
	"savesPath": "",
	"showFps": false,
	"useGameMode": false,
	"language": "",
	"wineVersion": {
		"bin": "/home/neros/.local/share/lutris/runners/wine/lutris-GE-Proton8-12-x86_64/bin/wine",
		"name": "Wine - lutris-GE-Proton8-12-x86_64",
		"type": "wine",
		"lib": "/home/neros/.local/share/lutris/runners/wine/lutris-GE-Proton8-12-x86_64/lib64",
		"lib32": "/home/neros/.local/share/lutris/runners/wine/lutris-GE-Proton8-12-x86_64/lib",
		"wineserver": "/home/neros/.local/share/lutris/runners/wine/lutris-GE-Proton8-12-x86_64/bin/wineserver"
	},
	"winePrefix": "/home/neros/Jeux/Heroic/compatdata/Beyond Blue",
	"wineCrossoverBottle": ""
}

Game launched at: Tue Jul 25 2023 02:57:27 GMT+0200 (heure d’été d’Europe centrale)

System Info:
Heroic Version: 2.9.0 Boa Hancock
Legendary Version:  0.20.32 Dark Energy (hotfix #6)
GOGdl Version: 0.7.3
Nile Version: 1.0.0 Jonathan Joestar

Electron Version: 24.4.1
Chrome Version: 112.0.5615.204
NodeJS Version: 18.14.0

OS: Arch KERNEL: 6.4.5-zen1-1-zen ARCH: x64
CPU: AMD Ryzen 5 3600 6-Core Processor @3.6 GOVERNOR: schedutil
RAM: Total: 31.27 GiB Available: 25.24 GiB
GRAPHICS: GPU0: AMD Radeon RX 6600 XT VRAM: 8176MB 
PROTOCOL: wayland

Launch Command: LD_PRELOAD= WINEPREFIX="/home/neros/Jeux/Heroic/compatdata/Beyond Blue" WINEDLLOVERRIDES=winemenubuilder.exe=d WINE_FULLSCREEN_FSR=0 WINEESYNC=1 WINEFSYNC=1 ORIG_LD_LIBRARY_PATH= LD_LIBRARY_PATH=/home/neros/.local/share/lutris/runners/wine/lutris-GE-Proton8-12-x86_64/lib64:/home/neros/.local/share/lutris/runners/wine/lutris-GE-Proton8-12-x86_64/lib GST_PLUGIN_SYSTEM_PATH_1_0=/home/neros/.local/share/lutris/runners/wine/lutris-GE-Proton8-12-x86_64/lib64/gstreamer-1.0:/home/neros/.local/share/lutris/runners/wine/lutris-GE-Proton8-12-x86_64/lib/gstreamer-1.0 WINEDLLPATH=/home/neros/.local/share/lutris/runners/wine/lutris-GE-Proton8-12-x86_64/lib64/wine:/home/neros/.local/share/lutris/runners/wine/lutris-GE-Proton8-12-x86_64/lib/wine /opt/heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch 3a0cae9455844390ad00505a704d641e --language fr --wine /home/neros/.local/share/lutris/runners/wine/lutris-GE-Proton8-12-x86_64/bin/wine --wrapper  

Game Log:
usage: legendary launch <App Name> [options]
legendary launch: error: argument --wrapper: expected one argument
```

### Steps to reproduce

1. Click on "play"
2. It crashes

### Expected behavior

Should launch the game

### Heroic Version

Latest Stable

### System Information

- OS: Arch

**Comment by arielj** [comment]
Sorry, we noticed the issue right after the release, we have the fix for it here https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/2893 (if you want to use that build for now).

Alternatives until that's released:
- use the build from the github PR
- use proton instead of wine (though the issue also affects native games so in those cases this option doesn't work)
- roll back to 2.8.0

**Comment by NerosTie** [comment]
Fixed in 2.9.1.
